src/Npgsql/NpgsqlTypes/NpgsqlDbType.cs

```diff
@@ -633,10 +633,6 @@ public static DbType ToDbType(this NpgsqlDbType npgsqlDbType)
             NpgsqlDbType.Char => DbType.String,
             NpgsqlDbType.Name => DbType.String,
             NpgsqlDbType.Citext => DbType.String,
-            NpgsqlDbType.Refcursor => DbType.Object,
```

Comment:
All of these are redundant, and we already don't explicitly map all the other cases of NpgsqlDbType either.